src/Utils/getFormattedNumber.ts

```diff
@@ -4,13 +4,19 @@ class NumberParser {
     const parts = format.formatToParts(12345.6);
     const numerals = Array.from({ length: 10 }).map((_, i) => format.format(i));
     const index = new Map(numerals.map((d, i) => [d, i]));
-    this._group = new RegExp(`[${parts.find(d => d.type === "group").value}]`, "g");
-    this._decimal = new RegExp(`[${parts.find(d => d.type === "decimal").value}]`);
+    this._group = new RegExp(
```

Comment:
Have you changed anything in this file or just reformatted it (like on another PR as well). It is extremely difficult to review the code if 90% is just reformatted without any change.

Reply:
It's just formatting. I haven't changed anything here.